src/hooks/useOrderForm.ts

```diff
@@ -26,16 +30,52 @@ export function useOrderForm(
   )
   const totalPrice = product.price.sellingPrice * totalQuantity
 
-  const onSubmit = (data: FormValues) => {
-    alert(`주문이 완료되었습니다.
-          상품명: ${product.name}
-          구매 수량: ${totalQuantity}
-          발신자 이름: ${data.sender}
-          메시지: ${data.message}`)
-    reset()
-    setSelectedCard(cardMock[0])
-    setValue('message', cardMock[0].defaultTextMessage)
-    setFinalReceivers([])
+  const { user } = useAuth()
+  const navigate = useNavigate()
+
+  const onSubmit = async (data: FormValues) => {
+    try {
+      await axios.post(
+        `${import.meta.env.VITE_API_BASE_URL}/api/order`,
+        {
+          productId: product.id,
+          message: data.message,
+          messageCardId: String(selectedCard.id),
+          ordererName: data.sender,
+          receivers: finalReceivers.map((r) => ({
+            name: r.name,
+            phoneNumber: r.phone,
+            quantity: Number(r.quantity),
+          })),
+        },
+        {
+          headers: {
+            Authorization: user?.authToken || '',
+          },
+        }
+      )
+      alert(`주문이 완료되었습니다.
+        상품명: ${product.name}
+        구매 수량: ${totalQuantity}
+        발신자 이름: ${data.sender}
+        메시지: ${data.message}`)
+      reset()
+      setSelectedCard(cardMock[0])
+      setValue('message', cardMock[0].defaultTextMessage)
+      setFinalReceivers([])
+      navigate('/')
+    } catch (error: any) {
+      const status = error.reponse?.status
+      const message = error.response?.data.data.message || '주문 실패'
+
+      if (status === 401) {
```

Comment:
응답코드는 axios에 정의되어 있는데요. 굳이 수정은 안해주셔도 되지만, 알아두시면 좋을 것 같아요!
```suggestion
      if (status === HttpStatusCode.Unauthorized) {
```